src/conn-scheduler.ts

```diff
@@ -178,10 +178,13 @@ export class ConnScheduler {
     return this.hops[data.key] === -1;
   };
 
-  private weFollowThem = ([_addr, data]: [string, {key?: string}]) => {
+  private weConnectToThem = ([_addr, data]: [string, {key?: string}]) => {
     if (!data?.key) return false;
     const h = this.hops[data.key];
-    return h > 0 && h <= 1;
+
+    // Peer with people we follow unless `config.hops` is set.
+    const maxHops = typeof this.config.hops === 'number' ? this.config.hops : 1
```

Comment:
Because this uses `config.hops`, this change is even more consequential than I thought it, because it will be a breaking change for every Manyverse users (Manyverse specifies `config.hops=2`). I was under the impression that we would use some other config, specially a config that is not shared with other modules. Libraries and users should have a way of specifying 2 different wishes: "how far in the social graph do I want to **replicate** feeds" (replication hops) and "how far in the social graph do I want to **connect** to peers" (connection hops).

Reply:
Also, because we're using TypeScript, this line could be written as `const maxHops = this.config.hops ?? 1`

Reply:
Oh, I was looking at some other bits of the code that referenced `config` and didn't see `config.conn`, so I assumed that the `config` variable was actually `config.conn`. I don't think this would conflict anyway because it's not `config.friends.hops`, but I'll push a commit fixing that.